clang/test/CodeGen/aarch64-sme2-intrinsics/acle_sme2_reinterpret_svcount_svbool.c

```diff
@@ -9,6 +9,12 @@
 
 #include <arm_sme.h>
 
+#if defined __ARM_FEATURE_SME
+#define MODE_ATTR __arm_streaming
+#else
+#define MODE_ATTR __arm_streaming_compatible
```

Comment:
Do you need to use `__arm_streaming_compatible` here?  Now we've agreed this keyword has no effect on the target features in use I think `MODE_ATTR` should remain blank to mirror the expected usage when SME is not in use.